cpp/src_prims/sparse/coo.cuh

```diff
@@ -831,9 +831,9 @@ void coo_symmetrize(COO<T> *in, COO<T> *out,
  * @param row_sizes: Input empty row sum 1 array(n)
  * @param row_sizes2: Input empty row sum 2 array(n) for faster reduction
  */
-template <typename math_t>
+template <typename math_t, typename value_idx>
```

Comment:
Nitpick: Can we change `math_t` to `value_t` and swap the params for consistency?